app/src/main/java/io/intrepid/contest/screens/conteststatus/ContestStatusContract.java

```diff
@@ -0,0 +1,14 @@
+package io.intrepid.contest.screens.conteststatus;
+
+import io.intrepid.contest.base.BaseContract;
+
+class ContestStatusContract {
+    interface View extends BaseContract.View {
+        void showWaitingSubmissionsFragment(int numSubmissionsMissing);
+
+        void showApiErrorMessage();
+    }
+
+    interface Presenter extends BaseContract.Presenter<View> {
```

Comment:
What is supposed to happen here? Maybe for now just a TImber log will suffice. Not sure though.